git-multimail/README

```diff
@@ -269,6 +269,25 @@ multimailhook.mailer
            The sender address to be passed to the SMTP server.  If
            unset, then the value of multimailhook.from is used.
 
+       multimailhook.smtpServerTimeout
+
+           Timeout in seconds.
+
+       multimailhook.smtpServerDebug
+
+           True/False. Set True to see all debug messages to/from
+           SMTP server.
+
+       multimailhook.smtpServerSecurity
+
+           Set the security type. Allowed values: none, ssl.
+           Default=none.
```

Comment:
It could be nice to allow customizing the port number too, in case users use a non-standard one. Don't feel forced to do it in this PR though, it can still be added later.